### PR TITLE
Changes to Arcade UI so it automatically forwards to VNC session

### PR DESCRIPTION
**Pull request by hjeeves**

### CODE REVIEW

File: _includes/_page_header.html [hunk 1]
```diff
@@ -91,12 +91,27 @@ <h5 class="my-0 mr-md-auto font-weight-normal">
...
+
+
+<script type="text/javascript">
+  $(document).ready(function () {
+    $("#cancel_login_button").click(function () {
+      parent.history.back()
+      return false
+    })
+  })
+</script>
```
Comment (at88mph) [inline _includes/_page_header.html]: There is no cancel button anymore.  This bit of script can be removed.

File: arcade/index_en.md [hunk 2]
```diff
@@ -82,37 +60,19 @@ permalink: /en/arcade/
...
 <!-- fontawesome *should* be included in the header -->
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+<link rel="stylesheet" href="/css/canfar.css">
+
+<script type="text/javascript" src="http://apps.canfar.net/cadcJS/javascript/org.opencadc.js"></script>
+<script type="text/javascript" src="http://apps.canfar.net/cadcJS/javascript/cadc.uri.js"></script>
+<script type="text/javascript" src="http://apps.canfar.net/canfar/javascript/cadc.user.js"></script>
+
+<script type="text/javascript" src="//www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/cadc/bootstrap/js/cadc.user.js"></script>
+<script type="text/javascript" src="//www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/cadc/bootstrap/js/login.js"></script>
```
Comment (at88mph) [inline arcade/index_en.md]: Please check the browser's `Network` tab to ensure none of these are being loaded twice; I think most, if not all, are loaded in the CANFAR site's header.  The CADC host cannot be used either as it breaks the NRC Internet Explorer 11 browser unfortunately.

File: _config.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ tagline: Processing and Storage on the cloud
 permalink: /news/:year/:month/:day/:title/
 relative_permalinks: false
 markdown: kramdown
-site_env: prod
+site_env: vm
```
[the remark this answers is not in the document]
Comment (hjeeves) [reply _config.yml]: Oh. how unfortunate. Thanks for flagging it.

File: _includes/_page_header.html (hunk 1 above)
Comment (hjeeves) [reply _includes/_page_header.html]: ok

**Pull request merged**